src/System.Private.Uri/src/System/Uri.cs

```diff
@@ -5367,7 +5367,7 @@ private static string PathDifference(string path1, string path2, bool compareCas
             // Same path except that path1 ended with a file name and path2 didn't
             if (relPath.Length == 0 && path2.Length - 1 == si)
                 return "./"; // Truncate the file name
-            return relPath.ToString() + path2.Substring(si + 1);
+            return relPath.Append(path2.AsSpan(si + 1)).ToString();
```

Comment:
Is there a significant difference between Substring() and AsSpan().ToString()?

Reply:
No... but this isn't AsSpan().ToString.  I think you're missing one of the parens in there :)

Reply:
Ooooh.  I was.  D'oh.